benches/gigagas.rs

```diff
@@ -7,10 +7,12 @@
 use std::{num::NonZeroUsize, thread};
 
 use ahash::AHashMap;
-use alloy_chains::Chain;
 use alloy_primitives::{Address, U160, U256};
 use criterion::{black_box, criterion_group, criterion_main, Criterion};
-use pevm::{execute_revm_parallel, execute_revm_sequential, EvmAccount, InMemoryStorage};
+use pevm::{
+    execute_revm_parallel, execute_revm_sequential, network::ethereum::PevmChainEthereum,
```

Comment:
`network::PevmEthereum` would be cleaner than `network::ethereum::PevmChainEthereum`.